.gitignore

```diff
@@ -8,6 +8,14 @@
 # Platforms
 .DS_Store
 
+# Package Manager/Conan
+*conanbuild.sh
+*conanbuildenv.sh
+*conanrun.sh
+*conanrunenv.sh
+*deactivate_conanbuild.sh
+*deactivate_conanrun.sh
+
 # Build System
 out/
 /src/test_driver/nrfconnect/build/
```

Comment:
not related to your PR but just noticed there is some stuff in here that is not related to this repo.